### PR TITLE
Django 2.0 compatibility

### PR DESCRIPTION
**Pull request by akx**

### CODE REVIEW

**Comment by akx** [comment]
Oofp, huh, seems this PR is very half-baked. Not sure what I was thinking submitting it yet. 😅 

Will get back to it later.

File: dfd_tests/settings.py [hunk 1]
```diff
@@ -19,20 +24,25 @@
...
+cms_apps = [
     'menus',
     'cms',
     'treebeard',
-    'form_designer',
     'form_designer.contrib.cms_plugins.form_designer_form',
+]
+
+INSTALLED_APPS = tuple(
+    [
```
Comment (suutari-ai) [inline dfd_tests/settings.py]: Just make it a list. A list like that being a tuple wasn't a good idea in the first place.

File: dfd_tests/urls.py [hunk 2]
```diff
@@ -1,6 +1,9 @@
 from django.conf.urls import include, url
 from django.contrib import admin
+import django
+
+admin_urls = (admin.site.urls if (django.VERSION[0] == 2) else include(admin.site.urls))
 
 urlpatterns = [
-    url(r'^admin/', include(admin.site.urls)),
+	url(r'^admin/', admin_urls),
```
Comment (suutari-ai) [inline dfd_tests/urls.py]: Why the indentation change?

**Comment by suutari-ai** [comment]
Very nice work @akx, thanks!

Left a couple comments. Can you check them out?

File: dfd_tests/urls.py (hunk 2 above)
Comment (akx) [reply dfd_tests/urls.py]: No idea. Must've been the editor, not me. 😇

**Comment by suutari-ai** [comment]
Thanks @akx!

**Pull request merged**